doc/TESTS.md

```diff
@@ -23,7 +23,7 @@ These tests use [rspec](http://rspec.info/) are not executed on travis, so you n
 
 **Run your tests always on your forked repo!**
 
-
+## Syntax variables
```

Comment:
This needs to another hash character so it can be a children of ```## Acceptance tests```

Reply:
wow beautiful explaination :) +!